server/channels/api4/plugin.go

```diff
@@ -20,7 +21,8 @@ import (
 )
 
 const (
-	MaximumPluginFileSize = 50 * 1024 * 1024
+	// Maximum number of bytes to hold in memory when reading plugin file, NOT the maximum plugin size
```

Comment:
Comments in go usually begin with the variable name
```suggestion
	// MaxPluginMemory is the maximum number of bytes to hold in memory when reading a plugin bundle.
```